bin/node/src/chain_spec.rs

```diff
@@ -93,6 +132,13 @@ impl ChainParams {
         self.base_path.clone().into()
     }
 
+    pub fn node_key_file(&self) -> &str {
+        match &self.node_key_file {
+            Some(filename) => filename,
+            None => "p2p_secret",
+        }
+    }
+
```

Comment:
Some fields have `default_value` specified in the attribute macro, and some in the getter. Can you add a ticket to unify this (preferrably specify the default value in the attribute macro)?

Reply:
https://cardinal-cryptography.atlassian.net/browse/A0-428